src/nupic/engine/RegionImplFactory.cpp

```diff
@@ -263,26 +244,10 @@ static RegionImpl * deserializePyNode(DynamicPythonLibrary * pyLib,
     std::string fullNodeType = std::string(package) + std::string(".") +
                                std::string(nodeType.c_str() + 3);
 
-    // Check if node exists and continue if not
-    std::string nodePath = Path::join(getPackageDir(pyLib->getRootDir(), package), 
-           std::string(nodeType.c_str() + 3) + std::string(".py"));
-
-    if (!Path::exists(nodePath))
```

Comment:
Since we just wanted the user to give the Python package, we wouldn't know the path.  That is why I eliminated that.  Knowing the path only eliminates unnecessary attempts at load a package.  If you try to load a package that doesn't exist, it returns null, so this isn't necessary.